### ISSUE TITLE
ran a small expressJs example with views - throwed "internal server Error" 

### ISSUE THREAD

**Issue by buzzshiva** [nodester/nodester#178]
Ran a small expressJs ( jade sytlus) example with views  but throwed 

http://apptest.nodester.com

"internal server Error" 

$express -t jade -c stylus
0 drwxr-xr-x+ 1 user None   0 Aug 11 17:31 public
1 -rw-r--r--  1 user None 180 Aug 11 17:31 package.json
0 drwxr-xr-x+ 1 user None   0 Aug 11 17:31 views
4 -rw-r--r--  1 user None 908 Aug 11 17:47 server.js

$ nodester logs
nodester info Showing logs for: apptest
New PID: 49044
chroot_runner
chroot_runner

$ nodester app info
nodester info Gathering information about: apptest
nodester info apptest on port 11053 running: true (pid: 49044)
nodester info gitrepo: git@nodester.com:/node/git/buzzshiva/2900-c4679e71e4c6842
361505c67399d7a6b.git
nodester info appfile: server.js

source code: Generated by express

/**
- Module dependencies.
  */

var express = require('express');

var app = module.exports = express.createServer();

// Configuration

app.configure(function(){
  app.set('views', __dirname + '/views');
  app.set('view engine', 'jade');
  app.use(express.bodyParser());
  app.use(express.methodOverride());
  app.use(require('stylus').middleware({ src: __dirname + '/public' }));
  app.use(app.router);
  app.use(express.static(__dirname + '/public'));
});

app.configure('development', function(){
  app.use(express.errorHandler({ dumpExceptions: true, showStack: true }));
});

app.configure('production', function(){
  app.use(express.errorHandler());
});

// Routes

app.get('/', function(req, res){
  res.render('index', {
    title: 'Express'
  });
});

app.listen(process.env['app_port']);

Ps:
Also How do I change default server.js to some other <filename.js> after creating the application using nodester.

**Comment by buzzshiva** [comment]
I think this might be the case ...

in my dev directory(apptest.nodester.com) ... I have two files starting with server_old.js and server_last.js;
Again the starting script is server.js;

I think (internal server error) in the nodester might be due to some regular expression that's looking for starting name "server"

In my situation there happens to be three files starting with server. Correct me if i'm wrong. I think this might be the 
case of greedy regex;

---

Solution:
When I have renamed the files back to _server_old.js and _server_last.js ; it started working normally.

http://apptest.nodester.com  

---

Updates:
I tested again with renaming the files back to starting word "server"  for _server_last.js and _server_old.js it worked normally. 

I still couldn't find why it throwed "Internal server Error" 

Is it because I skipped the first two steps for git when I created new files using express js framework;  

Step 1 :git init
Step 2 :git add .
Step 3 :git commit -m " new files "
Step 4: git push
